configs/components/_base-ruby-augeas.rb

```diff
@@ -4,6 +4,7 @@
 #
 
 pkg.add_source("file://resources/patches/augeas/ruby-augeas-0.5.0-patch_c_extension.patch")
+pkg.add_source("file://resources/patches/augeas/ruby-augeas-0.5.0-patch_remove_unused_parameter.patch")
```

Comment:
For this patch, will we at some point want to no longer apply this patch? IE, will Augeas eventually include the above patch? If so it'd be helpful to leave a comment or some reminder to our future selves that we can remove this patch.

Reply:
Thanks Christopher! Done.